tasks/configure.yml

```diff
@@ -7,8 +7,17 @@
     path: "{{ etc_prefix }}/haproxy/frontends.d"
     state: directory
 
-- name: 'Empty the folder if not already empty'
-  command: find {{ etc_prefix }}/haproxy/frontends.d -name *.cfg -exec rm -f {} \;
+- name: "List files for the frontends"
+  shell: ls -1 "{{ etc_prefix }}/haproxy/frontends.d"
+  register: directory_contents
+  changed_when: false
+
+- name: "Remove unmanaged files for the frontends"
+  file:
+    path: "{{ etc_prefix }}/haproxy/frontends.d/{{item}}"
```

Comment:
Please surround Jinja2 variables with spaces: `{{ item }}` instead of `{{item}}`.